dev/tests/acceptance/tests/functional/Magento/FunctionalTest/AdminNotification/composer.json

```diff
@@ -12,7 +12,7 @@
     },
     "require": {
         "magento/magento2-functional-testing-framework": "1.0.0",
-        "php": "7.0.2|7.0.4|~7.0.6|~7.1.0"
+        "php": "~7.1.3||~7.2.0"
```

Comment:
I just noticed that MFTF tests has been also updated. I think it's better to be done in a separate task.
MFTF tests are run separately from the Magento application, so they can still run on its own supported PHP version. In scope of updating MFTF dependencies we should verify that the framework is not broken, which increases scope for this task.

Reply:
Sounds good. I'll back those changes out.